iocage/Filter.py

```diff
@@ -156,21 +158,7 @@ def _filter_string_has_globs(self, filter_string: str) -> bool:
         return False
 
     def _split_filter_values(self, user_input: str) -> typing.List[str]:
-        values: typing.List[str] = []
-        escaped_comma_blocks = map(
-            lambda block: block.split(","),
-            user_input.split("\\,")
-        )
-        for block in escaped_comma_blocks:
-            n = len(values)
-            if n > 0:
-                index = n - 1
-                values[index] += f",{block[0]}"
-            else:
-                values.append(block[0])
-            if len(block) > 1:
-                values += block[1:]
-        return values
+        return re.split(r'(?<!\\),', user_input)
```

Comment:
wat

Reply:
Split by non-escaped comma.

Reply:
please add that as comment

Reply:
Moved it to a global variable - it's name steps in instead of a comment.